pandas/tests/test_style.py

```diff
@@ -0,0 +1,260 @@
+import copy
+from nose import SkipTest
+from unittest import TestCase
+
+import numpy as np
+import pandas as pd
+from pandas import DataFrame
+
+from pandas.core.style import Styler
+import pandas.util.testing as tm
+
+
+class TestStyler(TestCase):
+
+    def setUp(self):
+        np.random.seed(24)
+        self.s = DataFrame({'A': np.random.permutation(range(6))})
+        self.df = DataFrame({'A': [0, 1], 'B': np.random.randn(2)})
+        self.f = lambda x: x
+        self.g = lambda x: x
```

Comment:
skip of no jinja2 (but add a test that checks raising if you try df.style and don't have jinja2 installed)